ContactDetailsApi/ContactDetailsApi.csproj

```diff
@@ -17,7 +17,7 @@
     <PackageReference Include="AWSXRayRecorder.Handlers.AspNetCore" Version="2.7.2" />
     <PackageReference Include="AWSXRayRecorder.Handlers.AwsSdk" Version="2.8.2" />
     <PackageReference Include="FluentValidation.AspNetCore" Version="10.3.3" />
-    <PackageReference Include="Hackney.Core.Authorization" Version="1.78.0" />
+    <PackageReference Include="Hackney.Core.Authorization" Version="1.82.0-feat-add-ipwhite0005" />
```

Comment:
Im guessing this will be updated after the changes are pushed to lbh-core?

Reply:
+1 I'm guessing this is waiting for the package PR to be merged.

Reply:
Yep, updated now